research/src/pages/toolbar.proposal.mdx

```diff
@@ -0,0 +1,72 @@
+---
+menu: Proposals
+name: Toolbar (Editor's Draft)
+path: /components/toolbar
+pathToResearch: /components/toolbar.research
+---
+
+import ToolbaAnatomy from '../components/toolbar-anatomy'
+
+## Overview
+
+A `toolbar` is a container for grouping a set of controls, such as buttons, menubuttons, or checkboxes.
+
+## Use Cases
+
+A `Toolbar` groups visually a set of controls semantically algined and it can also be ane effective way of reducing the number of tab stops
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Stardust](https://fluentsite.z22.web.core.windows.net/0.59.0/components/toolbar/definition)
+- [Reakit](https://reakit.io/docs/toolbar/)
+- [Radix UI](https://www.radix-ui.com/docs/primitives/components/toolbar)
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+<ToolbaAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<toolbar>` - The root element that that has the semantic value of a toolbar **[required]**
+- `<toolbar-item>` - The item to be rendered in the toolbar **[required]**
+
+Proposed HTML:
+
+```html
+<div>
+  <button />
+  <div>
+    <button />
+    <button />
+  </div>
+  <span />
+  <button />
+</div>
+```
+
+#### Accessibility <a class="link" href="#accessibility" id="accessibility"></a>
+
+- The element that serves as the toolbar container has role toolbar.
+- If the toolbar has a visible label, it is referenced by aria-labelledby on the toolbar element. Otherwise, the toolbar element has a label provided by aria-label.
```

Comment:
This is how the ARIA practices is outlining how to use ARIA to provide a toolbar an accessible name.  Could not this be more open to possibilities?  for instance, rather than using `aria-labelledby`, if a `fieldset` were used as the base container with a `role=toolbar`, then it could leverage the `legend` element to specify a name for the toolbar.

the end goal of this could also be that HTML provides mechanisms to name certain elements.  This topic has come up with HTML for use of naming landmark elements without use of ARIA, or using the title attribute.  a toolbar would be another instance of where'd we need HTML to support this naming feature request.

Reply:
For a while now, I've wondered whether there was a way of adding `<label for="something">` to elements other than `<input>`. Scott's comment leads me to think mine wasn't an isolated thought.

Reply:
yeh, there's an issue on the HTML spec to do this... somewhere.  i'd have to dig.  but it's something that came up again recently per the recent request to add the `<search>` element.